packages/dev/core/src/Layers/highlightLayer.ts

```diff
@@ -79,15 +79,15 @@ class GlowBlurPostProcess extends PostProcess {
         });
     }
 
-    protected override async _initShaderSourceAsync(useWebGPU: boolean) {
+    protected override _gatherImports(useWebGPU: boolean, list: Promise<any>[]) {
         if (useWebGPU) {
             this._webGPUReady = true;
-            await import("../ShadersWGSL/glowBlurPostProcess.fragment");
+            list.push(import("../ShadersWGSL/glowBlurPostProcess.fragment"));
         } else {
-            await import("../Shaders/glowBlurPostProcess.fragment");
+            list.push(import("../Shaders/glowBlurPostProcess.fragment"));

```

Comment:
@RaananW I hope this is ok for our build?

Reply:
yes, as long as there is `import("...")` in one line it is fine